repositories.bzl

```diff
@@ -11,20 +11,20 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 # )
 IO_GRPC_GRPC_JAVA_ARTIFACTS = [
     "com.google.android:annotations:4.1.1.4",
-    "com.google.api.grpc:proto-google-common-protos:2.22.0",
+    "com.google.api.grpc:proto-google-common-protos:2.29.0",
     "com.google.auth:google-auth-library-credentials:1.4.0",
     "com.google.auth:google-auth-library-oauth2-http:1.4.0",
-    "com.google.auto.value:auto-value-annotations:1.10.2",
-    "com.google.auto.value:auto-value:1.10.2",
+    "com.google.auto.value:auto-value-annotations:1.10.4",
+    "com.google.auto.value:auto-value:1.10.4",
     "com.google.code.findbugs:jsr305:3.0.2",
     "com.google.code.gson:gson:2.10.1",
-    "com.google.errorprone:error_prone_annotations:2.20.0",
+    "com.google.errorprone:error_prone_annotations:2.23.0",
     "com.google.guava:failureaccess:1.0.1",
-    "com.google.guava:guava:32.0.1-android",
+    "com.google.guava:guava:32.1.3-android",
     "com.google.re2j:re2j:1.7",
     "com.google.truth:truth:1.1.5",
     "com.squareup.okhttp:okhttp:2.7.5",
-    "com.squareup.okio:okio:2.10.0",
+    "com.squareup.okio:okio:2.10.0",  # 3.0+ exposes jvm vs android to targets
```

Comment:
Not sure what to do with this comment. Does it mean we'd have to keep it on 2.x forever? We should probably leave a note when it's safe to upgrade.

Reply:
I've updated the comment to give a better clue what the hold-up is.